assets/styles/hero-carousel.scss

```diff
@@ -0,0 +1,45 @@
+.hero-carousel {
```

Comment:
This part is already in the `v3` branch and shouldn't be  part of this pull request.